backend/clubs/views.py

```diff
@@ -5027,20 +5032,40 @@ def initiate_checkout(self, request, *args, **kwargs):
             return Response(
                 {
                     "success": False,
-                    "detail": "Cart is stale, invoke /api/tickets/cart to refresh",
+                    "detail": (
+                        "Cart is stale or empty, invoke /api/tickets/cart to refresh"
+                    ),
+                    "sold_free_tickets": False,
                 },
                 status=status.HTTP_403_FORBIDDEN,
             )
 
         cart_total = self._calculate_cart_total(cart)
 
+        # If all tickets are free, we can skip the payment process
         if not cart_total:
+            order_info = {
+                "amountDetails": {"totalAmount": "0.00"},
+                "billTo": {
+                    "reconciliationId": None,
```

Comment:
Don't think we need this line.

Reply:
Please explain

Reply:
We're not using `reconciliationId` from `order_info` in `_give_tickets`.

Reply:
Yah, good catch. Wasn't clear which line you meant from my end